src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewTabs/StudentToolsTracesViewTabs.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { useTracesSummaryQuery } from '@/features/student/queries'
+import StudentToolsTracesViewUnassociatedTab
+  from '@/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewUnassociatedTab/StudentToolsTracesViewUnassociatedTab.vue'
+import { AvTab, AvTabs, MDI_ICONS } from '@avenirs-esr/avenirs-dsav'
+import { useI18n } from 'vue-i18n'
+
+const { t } = useI18n()
+const { data: tracesSummary } = useTracesSummaryQuery()
+
+const activeTab = ref(0)
+</script>
+
+<template>
+  <AvTabs v-model="activeTab">
+    <AvTab
+      :title="`${t('student.views.studentToolsTracesView.studentToolsTracesViewTabs.unassociated', tracesSummary?.unassociated ?? 0)}`"
```

Comment:
```suggestion
      :title="`${t('student.views.studentToolsTracesView.studentToolsTracesViewTabs.unassociated', {count: tracesSummary?.unassociated ?? 0})}`"
```
On avait pas vu qu'un truc comme ça était obligé ?

Reply:
Avec ce que tu as proposé juste au dessus c'est possible ducoup je vais regarder 🙂